scripts/speaker_tasks/pathsfiles_to_manifest.py

```diff
@@ -123,7 +147,7 @@ def main(wav_scp, text_scp=None, rttm_scp=None, uem_scp=None, ctm_scp=None, mani
     parser.add_argument("--paths2rttm_files", help="path to text file containing list of rttm files", type=str)
     parser.add_argument("--paths2uem_files", help="path to uem files", type=str)
     parser.add_argument("--paths2ctm_files", help="path to ctm files", type=str)
-    parser.add_argument("--manifest_filepath", help="scp file name", type=str, required=True)
+    parser.add_argument("--manifest_filepath", help="output manifest file name", type=str, required=True)
```

Comment:
could you change files name to file path to reduce confusion

Reply:
Changed to path to output manifestfile.

Reply:
Could we output manifest file path instead just to be accurate?